src/server/session.ts

```diff
@@ -108,17 +107,34 @@ module ts.server {
 
     export interface ServerHost extends ts.System {
     }
+    
+    export interface Environment {
+        byteLength: (buf: string, encoding?: string) => number;
+        hrtime: (start?: number[]) => number[]; //array of seconds, nanoseconds
+    }
+    
+    export interface Message {

```

Comment:
It is? Alright, I'll use it from there. In this file it was being stolen from node.d.ts's debug message format for some reason.